images/sriov-cni-daemonset.yaml

```diff
@@ -23,7 +23,8 @@ spec:
         effect: NoSchedule
       containers:
       - name: kube-sriov-cni
-        image: nfvpe/sriov-cni:latest
+        image: nfvpe/sriov-cni
+        imagePullPolicy: Never
```

Comment:
The default policy is `IfNotPresent` and for some reason it was still pulling image from external registry instead of using local image. And only way I could force it to use local image is by setting `imagePullPolicy: Never`. I don't mind really as long as we make sure we have the updated image in the external registry. But If  someone wants to build a latest image locally and run this Pod spec, it will not use the local image but instead pull one from external one. The `imagePullPolicy: Never`  of `:latest` tag will always pull image from external repo.

Reply:
I'm not aware of the `imagePullPolicy` problem, let's just stick to what it is now, and change later if necessary.